### ISSUE TITLE
requestFullscreen():  Promises must be handled appropriately

### ISSUE THREAD

**Issue by guoyunhe** [palantir/tslint#4190]
### Bug Report

- __TSLint version__: 5.11.0
- __TypeScript version__: 2.9.2
- __Running TSLint via__: (pick one) CLI / Node.js API

#### TypeScript code being linted

```ts
// code snippet
element.requestFullscreen()
```

with `tslint.json` configuration:

```json
{
  "extends": [
    "tslint-config-standard",
    "tslint-config-prettier",
    "tslint-no-circular-imports"
  ],
  "rules": {
    "no-console": [
      true,
      "log"
    ],
    "quotemark": [true, "single", "jsx-double"],
    "no-extra-boolean-cast": [0],
    "no-unused-variable": false
  }
}
```

#### Actual behavior
ERROR: 20:5  no-floating-promises  Promises must be handled appropriately

#### Expected behavior
It should not throw any errors. `requestFullScreen()` doesn't return promises.

**Comment by johnwiseheart** [comment]
`requestFullScreen` does return a promise according to the [MDN docs](https://developer.mozilla.org/en-US/docs/Web/API/Element/requestFullScreen)?

**Comment by guoyunhe** [comment]
But TypeScript's definition is `requestFullscreen(): void;`. If I append a `.then().catch()`, TypeScript throws errors. What should I do?

**Comment by guoyunhe** [comment]
Thanks! I disabled these lines. I think I need to check latest TypeScript 3.0 and make a bug ticket there.
